### PR TITLE
Basic connection to local db and insert of email data. 

### PR DESCRIPTION
**Pull request by jebbinSCL**
Connect with local dynamodb
Create table
Store first message if it doesn't exist.

### CODE REVIEW

File: Imap/Program.fs [hunk 1]
```diff
@@ -1,26 +1,97 @@
...
+    [<DynamoDBTable("Messages")>]
+    [<CLIMutable>]
+    type Message = { 
+        Id : string 
+        Subject : string
+        TextBody : string
+        HtmlBody : string
+        FromEmail : string
+        ToEmail : string
+        }
```
Comment (csscottlogic) [inline Imap/Program.fs]: This curly looks strange lined up vertically with the property names above?

File: Imap/Program.fs [hunk 2]
```diff
@@ -1,26 +1,97 @@
...
 
     let inbox = client.Inbox
     inbox.Open(FolderAccess.ReadOnly) |> ignore
 
     printfn "Total messages: %d" inbox.Count
     printfn "Recent messages: %d" inbox.Recent
 
-    let subjects = [0 .. inbox.Count - 1] |> List.map (fun i ->  inbox.GetMessage(i).Subject)
-    let print (subject: string) = Console.WriteLine(subject)
+    let messages = List.init inbox.Count (fun i -> inbox.GetMessage(i))
```
Comment (csscottlogic) [inline Imap/Program.fs]: ```List.init``` is much better than what I did with the range

**Comment by csscottlogic** [comment]
Can we have the refactoring you mentioned as part of the pull request please? Otherwise, we'll be reviewing things that will disappear or become more legible in the next PR.

**Comment by pallenSL** [comment]
Code-wise, we seem to go to some trouble to return Message.Id just to |> ignore it.
Otherwise it compiles, runs, and does nothing.  Is that good?  ;)

**Comment by pallenSL** [comment]
Fine (as far as it goes).
:ship:

**Comment by ianlovell** [comment]
:shipit:

**Pull request merged**